libs/vkd3d/command.c

```diff
@@ -1990,9 +1990,9 @@ static inline struct d3d12_command_list *impl_from_ID3D12GraphicsCommandList(d3d
     return CONTAINING_RECORD(iface, struct d3d12_command_list, ID3D12GraphicsCommandList_iface);
 }
 
-static void d3d12_command_list_invalidate_current_framebuffer(struct d3d12_command_list *list)
+static void d3d12_command_list_invalidate_rendering_info(struct d3d12_command_list *list)
 {
-    list->current_framebuffer = VK_NULL_HANDLE;
+    list->rendering_info.state_flags &= ~VKD3D_RENDERING_CURRENT;
```

Comment:
Does this mean we risk dropping calls to vkCmdEndRenderingKHR?

Reply:
Nah. CURRENT vs ACTIVE I suppose.

Reply:
We shouldn't ever drop any, removing `CURRENT` just notifies that we need to restart the render pass.